css/properties.json

```diff
@@ -4220,6 +4220,50 @@
     "status": "standard",
     "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/contain"
   },
+  "contain-intrinsic-size": {
+    "syntax": "[ none | <length> | auto <length> ]{1,2}",
+    "media": "visual",
+    "inherited": false,
+    "animationType": [
+      "contain-intrinsic-width",
+      "contain-intrinsic-height"
+    ],
+    "percentages": [
+      "contain-intrinsic-width",
+      "contain-intrinsic-height"
+    ],
+    "groups": [
+      "CSS Containment"
+    ],
+    "initial": [
+      "contain-intrinsic-width",
+      "contain-intrinsic-height"
+    ],
+    "appliesto": "allElements",
+    "computed": [
+      "contain-intrinsic-width",
+      "contain-intrinsic-height"
+    ],
+    "order": "perGrammar",
+    "status": "experimental",
+    "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/contain-intrinsic-size"
+  },
+  "contain-intrinsic-width": {
```

Comment:
This is from https://w3c.github.io/csswg-drafts/css-sizing-4/#propdef-contain-intrinsic-width

I'll do https://w3c.github.io/csswg-drafts/css-sizing-4/#propdef-contain-intrinsic-size when this part is "approved".

Reply:
Yes I do. I mean if this is about right, I'll add docs for other properties. Might add those now so it can all merge together.

Reply:
Done.